src/logcollector/read_win_event_channel.c

```diff
@@ -537,9 +550,10 @@ void win_start_event_channel(char *evt_log, char future, char *query)
     EVT_HANDLE bookmark = NULL;
     EVT_HANDLE result = NULL;
     int status = 0;
+    static int counter = 0;
```

Comment:
Same here: Logcollector is multithreading so using a static variable here is thread-unsafe unless it is protected by mutexes when writing and reading it. Let's protect that variable or look for another way to show the log just once.